### PR TITLE
Die when remote backend programs are missing

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
Testing, but PR open for feedback.

### CODE REVIEW

File: consoles/localXvnc.pm [hunk 1]
```diff
@@ -96,7 +96,8 @@ sub activate {
     sleep 1;
 
     # we need a window manager for fullscreen apps to work
-    system("DISPLAY=$display icewm -c $bmwqemu::scriptdir/consoles/icewm.cfg & echo \"icewm PID is \$!\"");
+    $icewm_return_code = system("DISPLAY=$display icewm -c $bmwqemu::scriptdir/consoles/icewm.cfg & echo \"icewm PID is \$!\"");
+    die('sh: icewm: command not found') if ($icewm_return_code == 32512);
```
Comment (Martchus) [inline consoles/localXvnc.pm]: You check for the exact return code `32512` here? I doubt that this return code will always be that arbitrary number. Why not checking `!= 0` instead?

Comment (coolo) [reply consoles/localXvnc.pm]: this is exit status 127 if you pass it through WEXITSTATUS :)

Comment (mudler) [reply consoles/localXvnc.pm]: maybe is safer to check if the binary is actually there before calling it or even  you can also use run_cmd https://github.com/os-autoinst/os-autoinst/search?q=runcmd&unscoped_q=runcmd

Comment (SergioAtSUSE) [reply consoles/localXvnc.pm]: $? == @127
But, system(...) == 32512

I didn't want to let it die on other cases.

**Comment by SergioAtSUSE** [comment]
What do you think now?

**Comment by okurz** [comment]
shouldn't be WIP I guess

**Comment by coolo** [comment]
Is this really the only program left to be checked? You better take the opportunity to review all tools used without check

**Comment by coolo** [comment]
then please:
- run ./tools/tidy
- squash your commits
- remove WIP from title

and we should be good to merge

**Comment by SergioAtSUSE** [comment]
@coolo, it should be ready.

The code coverage complaints. Should I do something about it?

**Pull request merged**